docs/get-started/secure-workflow.md

```diff
@@ -0,0 +1,263 @@
+    --tx-in 4e3a6e7fdcb0d0efa17bf79c13aed2b4cb9baf37fb1aa2e39553d5bd720c5c99#4 \
+    --tx-out $(cat payment2.addr)+0 \
+    --tx-out $(cat payment.addr)+0 \
+    --invalid-hereafter 0 \
+    --fee 0 \
+    --out-file tx.draft
+```
+
+#### Calculate the fee
+
+A simple transaction needs one input, a valid UTXO from `payment.addr`, and two outputs:
+
+* Output1: The address that receives the transaction.
+* Output2: The address that receives the change of the transaction.
+
+Note that to calculate the fee you need to include the draft transaction
+
+```sh
+cardano-cli transaction calculate-min-fee \
+    --tx-body-file tx.draft \
+    --tx-in-count 1 \
+    --tx-out-count 2 \
+    --witness-count 1 \
+    --byron-witness-count 0 \
+    --mainnet \
+    --protocol-params-file protocol.json
+```
+
+#### Calculate the change to send back to `payment.addr`
+
+All amounts must be in Lovelace:
+
+    expr <UTXO BALANCE> - <AMOUNT TO SEND> - <TRANSACTION FEE>
+
+For example, if we send 10 ada from a UTxO containing 20 ada, the change to send back to `payment.addr` after paying the fee is: 9.832035 ada:
+
+```sh
+expr 20000000 - 10000000 - 167965
+9832035
+```
+
+#### Build the transaction
+
+We write the transaction in a file, we will name it `tx.raw`.
+
+```sh
+cardano-cli transaction build-raw \
+    --tx-in 4e3a6e7fdcb0d0efa17bf79c13aed2b4cb9baf37fb1aa2e39553d5bd720c5c99#4 \
+    --tx-out $(cat payment2.addr)+10000000 \
+    --tx-out $(cat payment.addr)+9832035 \
+    --fee 167965 \
+    --out-file tx.raw
+```
+
+#### Sign the transaction
+
+Sign the transaction with the signing key `payment.skey` and save the signed transaction in `tx.signed`:
+
+```sh
+cardano-cli transaction sign \
+    --tx-body-file tx.raw \
+    --signing-key-file payment.skey \
+    --mainnet \
+    --out-file tx.signed
+```
+
+Save the `tx.signed` file back on the transfer memory stick, then [safely remove](https://help.ubuntu.com/stable/ubuntu-help/files-removedrive.html.en) the memory stick from the Air Gap machine.
+
+### 3. **Upload** and submit the Tx file.
+
+Reattach your transfer memory stick back to the Internet connected computer, then upload the `tx.signed` file to your Cardano node.
+
+#### Submit the transaction
+
+Log into your Cardano node and execute:
+
+```sh
+cardano-cli transaction submit \
+    --tx-file tx.signed \
+    --mainnet
+```
+
+Then check for a successful transaction by whatever means you prefer, e.g. as illustrated in [Check the balances](../stake-pool-course/handbook/create-simple-transaction#check-the-balances).
+
+## FAQ
+
+### Why can't I use `cardano-cli transaction build`?
+
+This is a convenient command to avoid the ["change" (return UTxO)
+calculation](../get-started/secure-workflow#calculate-the-change-to-send-back-to-paymentaddr)
+which requires you to prepare a test transaction,
+estimate fees, and calculate a final value of the funds to be moved.
+Instead, `transaction build` sends back "change" to a designated address.
```

Comment:
Looks like there were carriage return from your text editor added (might not be intentional?)